package.json

```diff
@@ -14,6 +14,7 @@
   "proxy": "http://localhost:4000",
```

Comment:
There should be a start script for the server too.

Reply:
Agree. Does `devStart` make sense? We are going to use it only in the development phase, so I thought it can be appropriate :computer: